.github/workflows/linux-ci.yml

```diff
@@ -23,11 +23,7 @@ jobs:
       with:
         fetch-depth: 0 # fetching all
 
-    - uses: actions/setup-dotnet@v1
```

Comment:
I think we need this. IIRC this issue is specific to linux.

Reply:
haha, yea I just recalled that you had explained this to me before. Seeing the problem first hand will serve to cement it in my 🧠.

Reply:
Maybe add a comment with a link to the issue if you have it